crates/arrow_tools/src/snapshots/arrow_tools__format__tests__pretty_format_markdown.snap

```diff
@@ -0,0 +1,31 @@
+---
+source: crates/arrow_tools/src/format.rs
+expression: formatted
+---
+# Item
+## content_chunk
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+Sed do eiusmod tempor aliqua. 
+
+reprehenderit nulla pariatur.
+
+## location
+path/to/folder/file_12345.txt
+
+## dist
+0.376276
+
+---
+# Item
+## content_chunk
+Lorem ipsum dolor adipiscing elit. 
+Cras venenatis euismod malesuada.
+
+## location
+path/to/folder/file_67890.txt
```

Comment:
These should just be attributes in the header. Everything outside the header is the content

Reply:
Like it 👍 - will update this

Reply:
Do you mean like this?
```markdown
---
location: 'path/to/folder/file_12345.txt'
dist: 0.376276
additional_column: 'foo'
---
Lorem ipsum dolor adipiscing elit. 
Cras venenatis euismod malesuada.

```

Reply:
@Jeadie  - yeah, similar to blogposts formatting
https://github.com/spiceai/docs/blob/trunk/website/blog/releases/v1.0.5.md?plain=1#L2

```md
---
date: 2025-03-11
title: 'Spice v1.0.5 (Mar 11, 2025)'
type: blog
authors: [sgrebnov]
tags: [release, iceberg, parquet, duckdb, arrow]
---

Announcing the release of Spice v1.0.5 🧊

Spice v1.0.5 expands Iceberg support with the introduction of the **Iceberg Data Connector**, in addition to the 
...
```

Reply:
Updated